### PR TITLE
Apply review suggestions 

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Apply review suggestions of #413

**DO NOT MERGE UNTIL RELEASE DATE UPDATED!!!**

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -292,7 +292,7 @@ manifest {
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
     version         = '2.1.0'
-    doi             = ''
+    doi             = 'https://doi.org/10.5281/zenodo.3240506'
```
Comment (bjlang) [inline nextflow.config:295]: I don't think this DOI makes sense. Shouldn't it be a pipeline specific one, like the one for RNAseq ([https://zenodo.org/records/13769441](https://zenodo.org/records/13769441))?

Comment (JoseEspinosa) [reply nextflow.config:295]: Is the pipeline specific one, but in the last release for some reason everything is wrong (authors, version number,...) I was going to try to fix it in the zenodo site, if possible.

Comment (bjlang) [reply nextflow.config:295]: Ah I see it now, thanks

**Review by bjlang — APPROVED** [review]
LGTM!

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**